src/utils/index.ts

```diff
@@ -51,18 +52,26 @@ export function getByteSize(data: string | Uint8Array): number {
   return Buffer.byteLength(data, 'utf-8')
 }
 
-export function secondsToDhms(secs: number): string {
+export function secondsToDhms(secs: number, resumed = false): string {
   const d = Math.floor(secs / (3600 * 24))
   const h = Math.floor((secs % (3600 * 24)) / 3600)
   const m = Math.floor((secs % 3600) / 60)
   const s = Math.floor(secs % 60)
 
-  const dDisplay = d > 0 ? d + (d === 1 ? ' day, ' : ' days, ') : ''
-  const hDisplay = h > 0 ? h + (h === 1 ? ' hour, ' : ' hours, ') : ''
-  const mDisplay = m > 0 ? m + (m === 1 ? ' minute, ' : ' minutes, ') : ''
+  const dDisplay = d > 0 ? d + (d === 1 ? ' day ' : ' days ') : ''
+  const hDisplay = h > 0 ? h + (h === 1 ? ' hour ' : ' hours ') : ''
+  const mDisplay = m > 0 ? m + (m === 1 ? ' minute ' : ' minutes ') : ''
   const sDisplay = s > 0 ? s + (s === 1 ? ' second' : ' seconds') : ''
 
-  return dDisplay + hDisplay + mDisplay + sDisplay
+  return resumed
+    ? dDisplay.length
+      ? dDisplay
+      : hDisplay.length
+      ? hDisplay
+      : mDisplay.length
+      ? mDisplay
+      : sDisplay
+    : dDisplay + hDisplay + mDisplay + sDisplay
```

Comment:
I am not sure about the name `resumed`, it does not tell me what it means without reading the code. Maybe call it `shortFormat`?

I propose this structure to avoid the nested ternaries which are a bit difficult to read at first

```
if (shortFormat) {
    // print only the most significant value
    return dDisplay || hDisplay || mDisplay || sDisplay
}

return dDisplay + hDisplay + mDisplay + sDisplay
```